src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,12 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from 'react'
+import Item from './Todo'
+
+const TodoList = props => {
+    return (
+        <div className='items'>
+            {props.items.map((item, i) => {return <Item item={item} click={props.click} id={i} key={i} />})}
```

Comment:
gets kind of rough when the word item is used so many times in a row, try giving things different more unique names to identify them better.